### PR TITLE
[Wallet] Support remote images for Firebase controlled notifications

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

This PR allows using remote images for Firebase controlled notifications.
This was needed as part of the work done for the Bidali integration.

It depends on #6510

### Other changes

- Removed local generic remote notification icon which only worked in dark mode
- Support layout without an icon too
- Cleaned up some outdated i18n strings

### Tested

**Example with Bidali icon set to a remote image**

**Example with no icon set**

### Related issues

- #6514

### Backwards compatibility

Yes

### CODE REVIEW

**Review by gnardini — APPROVED** [review]
Really cool to have this, love that this feature is becoming more useful 😄

File: packages/react-components/components/SimpleMessagingCard.tsx [hunk 1]
```diff
@@ -20,11 +20,22 @@ export default function SimpleMessagingCard({
...
+        style={
+          Object.prototype.hasOwnProperty.call(iconProp, 'uri') ? styles.remoteIcon : undefined
+        }
+        testID={`${testID}/Icon`}
+      />
+    )
   ) : (
-    // @ts-ignore isValidElement check above ensures image is an image source type
-    <Image source={iconProp} resizeMode="contain" />
+    undefined
```
Comment (tarikbellamine) [inline packages/react-components/components/SimpleMessagingCard.tsx:38]: Worthwhile to have a default icon for when one isn't provided?

File: packages/react-components/components/SimpleMessagingCard.tsx [hunk 2]
```diff
@@ -20,11 +20,22 @@ export default function SimpleMessagingCard({
   callToActions,
   testID,
 }: Props) {
-  const icon = React.isValidElement(iconProp) ? (
-    iconProp
+  const icon = iconProp ? (
+    React.isValidElement(iconProp) ? (
+      iconProp
```
Comment (tarikbellamine) [inline packages/react-components/components/SimpleMessagingCard.tsx:25]: What happens if a link breaks?

File: packages/mobile/scripts/sync_branding.sh [hunk 3]
```diff
@@ -21,7 +21,7 @@ echo $mobile_root
 cd "$mobile_root"
 
 # Please update the sha when valora branding updates are needed
-valora_branding_sha=74d23db
+valora_branding_sha=c678332
```
Comment (tarikbellamine) [inline packages/mobile/scripts/sync_branding.sh:24]: Why did Valora branding need to change for this PR?

**Review by tarikbellamine — APPROVED** [review]
Good stuff!

File: packages/react-components/components/SimpleMessagingCard.tsx (hunk 1 above)
Comment (jeanregisser) [reply packages/react-components/components/SimpleMessagingCard.tsx:38]: Yes I thought about it then decided not to do it as this gives us flexbility to intentionally not display an icon.

File: packages/react-components/components/SimpleMessagingCard.tsx (hunk 2 above)
Comment (jeanregisser) [reply packages/react-components/components/SimpleMessagingCard.tsx:25]: The space allocated for the image would remain empty.

File: packages/mobile/scripts/sync_branding.sh (hunk 3 above)
Comment (jeanregisser) [reply packages/mobile/scripts/sync_branding.sh:24]: I removed `remoteNotification` (moved remotely) which was in the Valora branding.

**Comment by Lss-Ankit** [comment]
Hi @jeanregisser  Verified task using Android play store build  v1.9.0(1004294331) & Test Flight Build V1.9.0(41) and found the following: 

- CTA Now Available: Buy Gift.... is not shown in Mainnet even after creating a new accounts. 
- Observed that CTA is shown into alfajore network build v1.9.0 (41) in IOS, but blank screen is shown after tapping on Buy Now button. 
Can you please let me know should i raised a ticket for the Alfajore network and do the needful that how we can see the CTA in Mainnet or let us know if we need to file a bug for the same. 

Attachment: [IOS_Alfajore_CTA_Issue.mp4](https://drive.google.com/file/d/1JuPVg_QHPx_KJL78IoCx7cAqoMEsz-9l/view?usp=sharing)

**Pull request merged**